odl/tomo/util/testutils.py

```diff
@@ -11,17 +11,25 @@
 from future import standard_library
 standard_library.install_aliases()
 
-import pytest
-
-
 __all__ = ('skip_if_no_astra', 'skip_if_no_astra_cuda', 'skip_if_no_skimage')
 
-
-skip_if_no_astra = pytest.mark.skipif("not odl.tomo.ASTRA_AVAILABLE",
-                                      reason='ASTRA not available')
-
-skip_if_no_astra_cuda = pytest.mark.skipif("not odl.tomo.ASTRA_CUDA_AVAILABLE",
-                                           reason='ASTRA CUDA not available')
-
-skip_if_no_skimage = pytest.mark.skipif("not odl.tomo.SKIMAGE_AVAILABLE",
-                                        reason='skimage not available')
+try:
+    import pytest
+except ImportError:
+    # Make trivial decorator
+    from odl.util import OptionalArgDecorator
+
+    class ident(OptionalArgDecorator):
```

Comment:
Why would we need to overwrite this, shouldn't this be the default implementation?